bears/general/FilenameBear.py

```diff
@@ -74,12 +100,13 @@ def run(self, filename, file,
         if ignore_uppercase_filenames and filename_without_extension.isupper():
             return
 
-        if new_name != filename_without_extension:
-            diff = Diff(file, rename=os.path.join(head, new_name + extension))
+        if messages:
+            diff = Diff(file, rename=os.path.join(
+                head, new_name + extension))
```

Comment:
is the linebreak necessary here? Would keep the old line without a linebreak:
```
diff = Diff(file, rename=os.path.join(head, new_name + extension))
```
One change less to review ;)